snapshots/lcow/lcow.go

```diff
@@ -321,7 +321,21 @@ func (s *snapshotter) createSnapshot(ctx context.Context, kind snapshots.Kind, k
 			return nil, err
 		}
 
-		scratchSource, err := s.openOrCreateScratch(ctx)
+		var snapshotInfo snapshots.Info
+		for _, o := range opts {
+			o(&snapshotInfo)
+		}
+
+		var sizeGB uint
+		if sizeGBstr, ok := snapshotInfo.Labels["io.microsoft.container.storage.rootfs.size-gb"]; ok {
```

Comment:
Same here. I think we need to use the naming from: https://github.com/containerd/containerd/pull/3055/commits/75f183887a99de2fd29d3144e5f9a1e09a069a67#diff-491f07c28eb851b720f11ce0d3dbb4f8R38